### PR TITLE
feature: ZENKO-845 Object monitor throughput test

### PR DESCRIPTION
**Pull request by bennettbuchanan**

### CODE REVIEW

File: tests/zenko_e2e/backbeat/tests/api/objectMonitor.js [hunk 1]
```diff
@@ -44,33 +59,23 @@ describe('API Route: /_/backbeat/api/metrics/crr/<site-name>/progress/' +
...
+            };
+            return getAndCheckResponse(path, expectedBody, next);
         },
     ], done));
 
     it('should monitor part uploads of an MPU object', done => waterfall([
         next => scalityUtils.completeMPUAWS(srcBucket, key, 50, next),
         (data, next) => {
-            const path = `/_/backbeat/api//metrics/crr/${destLocation}` +
+            const path = `/_/backbeat/api/metrics/crr/${destLocation}` +
```
Comment (philipyoo) [inline tests/zenko_e2e/backbeat/tests/api/objectMonitor.js]: 😓 good catch, sorry for the typos

**Review by dora-korpar — APPROVED** [review]

**Review by philipyoo — APPROVED** [review]

**Review by tmacro — APPROVED** [review]

File: tests/zenko_e2e/backbeat/tests/api/objectMonitor.js (hunk 1 above)
Comment (bennettbuchanan) [reply tests/zenko_e2e/backbeat/tests/api/objectMonitor.js]: 🙈

**Review by ssalaues — APPROVED** [review]

**Review by rahulreddy — APPROVED** [review]

**Comment by ssalaues** [comment]
Lets wait on #219 before merging so we can verify that it runs in CI

**Comment by ssalaues** [comment]
If you rebase, the new fixes from #224 should help with the CI builds

**Pull request merged**